packages/neuron-wallet/src/utils/signWitnesses.ts

```diff
@@ -0,0 +1,46 @@
+import { bytes, number } from '@ckb-lumos/codec'
+import { serializeWitnessArgs } from './serialization'
+import { CKBHasher } from '@ckb-lumos/base/lib/utils'
+import { key } from '@ckb-lumos/hd'
+
+type StructuredWitness = CKBComponents.WitnessArgs | CKBComponents.Witness
+
+// https://github.com/nervosnetwork/ckb-system-scripts/wiki/How-to-sign-transaction#signing
+export const signWitnesses = ({
```

Comment:
Will the Lumos support sign a transaction?

Reply:
Lumos supports signing transaction: [refer link](https://lumos-website.vercel.app/#5-create-a-transfer-transaction), but just accept `TransactionSkelenton` or `Transaction` types, here the usage in Neuron is passing in a transaction-hash and witnesses, so we can't directly use lumos to sign it here.